Azure Services/Azure Monitor/Agents/Migration Tools/DCR Config Generator/VMIDcrGeneratorModule/VMIDcrGeneratorModule.psm1

```diff
@@ -20,18 +20,18 @@ function Get-VmiDataSources
     
     $vmiPerfCounter = [ordered]@{
         "name" = "VMInsightsPerfCounters";
-        "streams" = "Microsoft-InsightsMetrics";
+        "streams" = @("Microsoft-InsightsMetrics");
         "scheduledTransferPeriod" = "PT1M";
         "samplingFrequencyInSeconds" = 60;
-        "counterSpecifiers" = "\\VmInsights\\DetailedMetrics";
+        "counterSpecifiers" = @("\\VmInsights\\DetailedMetrics");
     }
     $vmiPerfCounters = @($vmiPerfCounter)
     $vmiDataSources = [ordered]@{
         "performanceCounters" = $vmiPerfCounters;
     }
     if ($ProcessAndDependencies) {
         $vmiExtension = [ordered]@{
-        "streams" = "Microsoft-ServiceMap";
+        "streams" = @("Microsoft-ServiceMap");

```

Comment:
https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/3b875bf3-0eec-4d8c-bdee-25c7ccc1f130/resourceGroups/vpidatala-rg/providers/Microsoft.Insights/dataCollectionRules/MSVMI-3/overview

Reference DCR:  streams need to be enclosed in an array[] , functionally this had no effect.